win32ss/gdi/gdi32/objects/dc.c

```diff
@@ -1057,6 +1057,10 @@ SetBkMode(
     _In_ HDC hdc,
     _In_ int iBkMode)
 {
+    /* Avoid bad mode setting */
+    if(iBkMode!=TRANSPARENT)
+        iBkMode=OPAQUE;
```

Comment:
Please add the spaces as well too, to make the code more readable.
```suggestion
    if(iBkMode != TRANSPARENT)
        iBkMode = OPAQUE;
```